README.md

```diff
@@ -2,92 +2,82 @@
+1. Tag augur-core as needed for a versioned deploy, e.g for a pre-release deployment: `npm version prerelease`
+2. CI will Build, Test, and Deploy the contracts
+3. augur-contracts will automatically have a new dev-channel version published to NPM (augur-contracts@dev)
+4. augur.js will automatically have a new dev-channel version published to NPM (augur.js@dev)
+5. If this is a real release (i.e. we want to declare this fit for public consumption), augur-contracts and augur.js should be published manually to NPM with their next version numbers.
+6. If this is a real release, augur-contracts and augur.js should be incremented to their NEXT release version number, and have the pre-release version -0 appended to them, and pushed to master.
 
-### Build:
+### Local deployment
 
-```
-npm run docker:build
-```
+Deployment can be run in two modes, direct or Docker. In direct mode, the assumption is that the entire system has been built (TypeScript and Solidity), and there is a working Node.js environment locally. Furthermore, one must possess an account on the deployment target (e.g. Rinkeby testnet) which has enough ETH to cover the costs of the deployment. For the purposes of deploying to the testnets, those with access to the augur private testnet keys can deploy and update the existing contracts. For reference, these keys are stored in an encrypted git repository within the Augur Project's keybase team. If you need access, please inquire in the Augur Discord.
 
-### Test:
+All deployment commands can be managed through scripts in package.json, and are accessible to `npm run` and `yarn`.
 
-```
-# With a pre-built image
-npm run docker:run:test:unit:all
+### Manual Rinkeby deployment
 
-# Build and run all unit tests and integration tests
-npm run docker:test
+Build and create compile artifacts:
 
-# Build and run just integration test
-npm run docker:test:integration
+```bash
+augur-core % npm run build
 ```
 
-### Debug:
+Deploy to Rinkeby:
 
+```bash
+augur-core % RINKEBY_PRIVATE_KEY=$(cat path/to/keys/deploy_keys/rinkeby.prv) npm run deploy:rinkeby
 ```
-npm run docker:run:test:console
-py.test -s tests/trading_tests.py
-```
-
-## General information about Augur
 
-- [A Roadmap For Augur and What’s Next](https://medium.com/@AugurProject/a-roadmap-for-augur-and-whats-next-930fe6c7f75a)
-- [Augur Master Plan](https://medium.com/@AugurProject/augur-master-plan-42dda65a3e3d)
-
-## Terminology
+### Updating augur-contracts
 
-[Augur Terminology](http://blog.augur.net/faq/all-terms/)
+Manual deployment generates artifacts in the `output/contracts` directory.  To merge these changes into augur-contracts, there are scripts located in [the augur-contracts repository](https://github.com/AugurProject/augur-contracts). **In local deployments, this is not automatic.**
 
-## Src Folders
+To merge the local changes into the augur-contracts repository:
```

Comment:
I'm generally against including manual deployment instructions in readmes.  No one should _ever_ be doing a manual deploy, so there is no need to include instructions.  If CI is broken, fix CI, don't give up and manually deploy (dangerous).  If for some unfathomable reason we _must_ deploy manually, a person can deduce the process from the `.travisci` file.